Makefile.toml

```diff
@@ -102,6 +102,13 @@ alias = "lqs-tests-flow"
 ################
 # Custom tasks #
 
+[tasks.check-all-format]
+workspace = false
+dependencies = [
+    "check-format",
+    "check-js-format",
+]
+
 [tasks.check-format]
```

Comment:
:question: Do we maybe want to rename this to `check-rs-format` for consistency reasons?